how-to/customize-workspace/client/src/modules/endpoints/fdc3-app/endpoint.ts

```diff
@@ -67,6 +68,7 @@ export async function requestResponse(
 			const passedApp: AppDefinitionTwoPointZero = applications[i] as AppDefinitionTwoPointZero;
 			const platformApp: PlatformApp = {
 				appId: passedApp.appId,
+				name: passedApp.name,
 				title: passedApp.title || passedApp.name,
```

Comment:
`??`

Reply:
best to discuss over voice, but the fdc3 open api can be passed a name. An alternative could be to extend App just in the open signature and use appId or name against our App.appid (if appId is empty in the mappers as fdc3 has name as the mandatory property for an app).

Reply:
Me being lazy, I mean use the `??` operator :)

Reply:
lol but I think it raises a good point on should we include name at all if it is just needed here.